Configure

```diff
@@ -10445,6 +10445,15 @@ int main()
 	else
 	    checkit("1e+34", buf);
 
+        /* Test for an Ubuntu/Debian bug in gcvt and qgcvt. See:        *
+         * https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1899553 */
+
+	Gconvert((DOUBLETYPE)0.4, 53, 0, buf);
+        if(sizeof(DOUBLETYPE) == 8)
+	  checkit("0.40000000000000002220446049250313080847263336181640625", buf);
+        else
+	  checkit("0.40000000000000000000542101086242752217003726400434971", buf);
+
```

Comment:
This looks very non-portable.  Given the problem being fixed should we just be checking the length of the resulting string?

Reply:
I think the lack of portability is not an issue wrt -Dusequadmath builds because they never call Gconvert() anyway.
However, I s'pose we should do the right thing by Double-Double and IEEE-754 128 bit long double builds - and, as you've suggested, that's not currently being done.

IMO, checking the length is a good enough place to start.
I don't know of any cases where the length will be correct && the value will be incorrect.
However, if such cases do exist, then a t/op/sprintf2.t failure should alert us to the fact.

I'll come up with something tomorrow.